cmd/srcd/cmd/prune_test.go

```diff
@@ -0,0 +1,65 @@
+// +build integration
+
+package cmd
+
+import (
+	"context"
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
+
+	cmdtest "github.com/src-d/engine/cmd/test-utils"
+
+	"github.com/stretchr/testify/suite"
+)
+
+// This test suite does not test the `--with-images` flag.
+// If the tests are run with `make test-integration` the daemon image will be
+// the one build locally (e.g. srcd/cli-daemon:dev-b72f1fe), and deleting
+// this image would make all the other tests fail.
+
+type PruneTestSuite struct {
+	cmdtest.IntegrationSuite
+	testDir string
+}
+
+func TestPruneTestSuite(t *testing.T) {
+	s := PruneTestSuite{}
+	suite.Run(t, &s)
+}
+
+func (s *PruneTestSuite) SetupTest() {
+	var err error
+	s.testDir, err = ioutil.TempDir("", "prune-test")
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func (s *PruneTestSuite) TearDownTest() {
+	s.RunStop(context.Background())
+	os.RemoveAll(s.testDir)
+}
+
+func (s *PruneTestSuite) TestRunningContainers() {
+	require := s.Require()
+
+	_, err := s.RunInit(context.TODO(), s.testDir)
+	require.NoError(err)
+
+	_, err = s.RunSQL(context.TODO(), "SELECT 1")
+	require.NoError(err)
+
+	_, err = s.RunCommand(context.TODO(), "prune")
```

Comment:
The integration tests for stop command use [`IntegrationSuite:RunStop()`](https://github.com/src-d/engine/pull/350/files#diff-6fd0a789630fde34454c1121b58bdf57R51); for consistency, this test could use a new `IntegrationSuite:RunPrune()`

Reply:
Honestly, I dislike methods like this
```go
func (s *IntegrationSuite) RunStop(ctx context.Context) (*bytes.Buffer, error) {
	return s.RunCommand(ctx, "stop")
}
```

And I would prefer to not add any new ones in this PR.

This is something we can discuss for #347, to either remove them all or add missing ones (like `RunPrune`).

Reply:
I think it's better to look for consistency in all iterations, bec then it's easier to clean later, but fair enough; I added it to my wishlist @ https://github.com/src-d/engine/issues/347#issuecomment-474337757